app/services/movie_service.rb

```diff
@@ -14,6 +18,12 @@ def search_movies(search_query)
     build_movie_objects(response)
   end
 
+  def movie
+   movie_params = JSON.parse(service.get("/3/movie/#{@url}").body, symbolize_names: true)
+   movie_params = movie_params.merge(JSON.parse(service.get("/3/movie/#{@url}/credits").body, symbolize_names: true))
+   movie_params = movie_params.merge(JSON.parse(service.get("/3/movie/#{@url}/reviews").body, symbolize_names: true))
+  end
```

Comment:
A potential refactor here would be to pull each of these API calls out into their own respective `private` methods, ie `get_credits`, `get_reviews`, `get_movie` and then the `movie` method would be responsible for merging all 3 responses and returning the result of that merge.

Reply:
Agreed, I'll make a note.